lib/gpu/host_buffer.cc

```diff
@@ -13,7 +13,8 @@ namespace gpu {
 
 IMPLEMENT_WRAPPERTYPEINFO(flutter_gpu, HostBuffer);
 
-HostBuffer::HostBuffer() : host_buffer_(impeller::HostBuffer::Create()) {}
```

Comment:
Oops, need to fix this too.

Reply:
@bdero how do I get access to the impeller context so I can grab an allocator for this?